release-notes/2.1/2.1.1-commit.md

```diff
@@ -35,4 +36,47 @@ This is a somewhat currated list of commits across the entire .NET Core 2.1.1 de
 
 * [`[0a99dd88]`](https://github.com/dotnet/wcf/commit/0a99dd88) Add net461 as a supported framework for S.SM.Security.
 * [`[45855085]`](https://github.com/dotnet/wcf/commit/45855085) Generate ThisAssembly.cs, update the version and links for svcutil.xmlserializer (#2893)
-* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
\ No newline at end of file
+* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
+
+## ASP
+
+### Entity Framework
+
+* [`[5bf9cc37]`](https://github.com/aspnet/EntityFrameworkCore/commit/5bf9cc370699f86e3609c183b3b23bfe2e3b1b23) Adds recursion guards to analyzer. [(#12054)](https://github.com/aspnet/EntityFrameworkCore/issues/12054)
+* [`[96049284]`](https://github.com/dotnet/corefx/pull/29846/commits/96049284dd577863c8b361fea06eadcfe5f1588b) Fix Sockets hang caused by concurrent Socket disposal [(#11993)](https://github.com/aspnet/EntityFrameworkCore/issues/11993)
```

Comment:
@leecow Two of these are commits from corefx--commits for EF issues are in the EF repo.

The commit for 11993 is: https://github.com/aspnet/EntityFrameworkCore/pull/12000/commits/a4a414432b20ae68ce53e4b981ecf9e718f98731

The commit for 11977 is: https://github.com/aspnet/EntityFrameworkCore/pull/12123/commits/9409a1f25b7603bb8dce747aff2a7df5744f5efe